symplyphysics/laws/electricity/maximum_moment_of_magnetic_field.py

```diff
@@ -0,0 +1,40 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The magnetic moment is the main physical quantity characterizing the magnetic properties of a substance,
+## that is, the ability to create and perceive a magnetic field.
+
+## Law is: m = I * S, where
+## m - magnetic moment,
+## I - current,
+## S - area of the circuit through which the current flows.
```

Comment:
Could you explain area more detailed?
Is it area of surface? Area of some loop from a wire?